src/mlvl_wrapper.rs

```
@@ -63,6 +63,29 @@ impl<'r, 'mlvl, 'cursor, 'list> MlvlArea<'r, 'mlvl, 'cursor, 'list>
         x.kind.as_mrea_mut().unwrap()
     }
 
+    pub fn toggle_memory_relay(&mut self, mem_relay_id: u32, active: u8)
+    {
+        let layer_id = ((mem_relay_id >> 26) & 0x1f) as usize;
+
+        for mem_relay in self.memory_relay_conns.iter_mut() {
```

Please panic if the memory relay is not found

---

Done